bookclub-backend/models/books.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const booksSchema = new mongoose.Schema({
+  gid: String,
```

Comment:
in order to prevent any book from being added more than once, this should probably be set to `{type: String, unique: true}`